snapcraft/_store.py

```diff
@@ -815,17 +873,20 @@ def _check_validations(validations):
         raise RuntimeError()
 
 
-def _sign_validation(validation, assertion, key):
+def _sign_assertion(target, assertion, key, endpoint):
     cmdline = ['snap', 'sign']
     if key:
         cmdline += ['-k', key]
     snap_sign = Popen(
         cmdline, stdin=subprocess.PIPE, stdout=subprocess.PIPE,
         stderr=subprocess.PIPE)
     data = json.dumps(assertion).encode('utf8')
-    logger.info('Signing validation {}'.format(validation))
+    logger.info('Signing {} assertion for {}'.format(endpoint, target))
```

Comment:
Good catch. That was different in validation vs snap developers. Fixed now. Thanks for the spot.